src/renderer/main/panels/configuration/Configuration.store.js

```diff
@@ -458,7 +457,7 @@ function create_configuration_manager() {
   }
 
   user_input.subscribe((ui) => {
```

Comment:
Creating a store subscription within a store can result in unexpected store triggers, such subscriptions are considered bad practice in "hot code path" and **doesn't have unsubscription** called anywhere.. Code is hard to understand to me, if the `createConfigListFrom(ui)` is generated in a *reactive way* within a store itself (which should be first called somewhere in a UI component), or it's triggered / called / updated externally.

I think it's worth discovering how the `user_input` subscription could be handled differently here, if we should create a `derived()` svelte store or use some other practice to create "clever svelte objects".

Reply:
Resolved. Subscription is reworked into a derived store.